.github/workflows/tests.yaml

```diff
@@ -0,0 +1,48 @@
+name: gabbi tests
+on:
+    push:
+        branches:
+        - master
+        - excision
```

Comment:
does this becomes solely `on: push` since presumably tests should run basically always?

Or is there some kind of more restricted behavior like "only on pull requests" or something that is more the norm?

Reply:
aye, will do